rest-service/manager_rest/rest/resources_v3_1/nodes.py

```diff
@@ -1,40 +1,43 @@
+from collections import defaultdict
+
 from ..resources_v3 import Nodes as v3_Nodes
+from ..resources_v2 import NodeInstances as v2_NodeInstances
 
 from manager_rest import manager_exceptions
 from manager_rest.rest import rest_utils
 from manager_rest.security.authorization import authorize
 from manager_rest.storage import get_storage_manager, models
 
 
+def _common_deployment_id(raw_nodes):
```

Comment:
moving this function out of the nodes endpoint, because it's useful in both nodes and node-instances

Reply:
but i'll soon replace this with `client.nodes.create_many(deployment_id, nodes)` anyway